rpc/block.go

```diff
@@ -326,8 +327,12 @@ func adaptBlockHeader(header *core.Header) BlockHeader {
 		Timestamp:        header.Timestamp,
 		SequencerAddress: sequencerAddress,
 		L1GasPrice: &ResourcePrice{
-			InWei: header.GasPrice,
-			InFri: nilToZero(header.GasPriceSTRK), // Old block headers will be nil.
+			InWei: header.L1GasPriceETH,
+			InFri: nilToZero(header.L1GasPriceSTRK),
+		},
+		L2GasPrice: &ResourcePrice{
+			InFri: nilToZero(header.L2GasPriceSTRK),
+			InWei: nilToZero(header.L2GasPriceETH),
```

Comment:
nit: please use the same order for `InWei` and `InFri` so that readers can read smoothly